packages/database/schema.prisma

```diff
@@ -509,6 +509,7 @@ model User {
   name                      String?
   email                     String           @unique
   emailVerified             DateTime?        @map(name: "email_verified")
+  image                     String?
```

Comment:
since this is a url, can we change this to `imageUrl` to make it clearer to the developer what to expect?